### ISSUE TITLE
don't show a progress bar when running in --non-interactive

### ISSUE THREAD

**Issue by fiws** [firebase/firebase-tools#73]
running `firebase --non-interactive --token "$FIREBASE_TOKEN" deploy` errors in my ci. (gitlab ci)

```
FIREBASE WARNING: Exception was thrown by user callback. TypeError: this.stream.clearLine is not a function
    at ProgressBar.terminate (/cache/project-www/node_modules/firebase-tools/node_modules/progress/lib/node-progress.js:177:17)
    at ProgressBar.tick (/cache/project-www/node_modules/firebase-tools/node_modules/progress/lib/node-progress.js:91:10)
    at /cache/project-www/node_modules/firebase-tools/lib/deploy/hosting/deploy.js:49:15
    at /cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:46:642
    at Bb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:43:165)
    at xb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:22:216)
    at yb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:21:1260)
    at Yh.g.Eb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:206:287)
    at Kh.g.Gd (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:193:283)
    at yh.Gd (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:183:364) 

Error: An unexpected error has occurred.
✔  2015 files uploaded successfully
FIREBASE WARNING: Exception was thrown by user callback. TypeError: this.stream.clearLine is not a function
    at ProgressBar.terminate (/cache/project-www/node_modules/firebase-tools/node_modules/progress/lib/node-progress.js:177:17)
    at /cache/project-www/node_modules/firebase-tools/lib/deploy/hosting/deploy.js:54:15
    at /cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:46:642
    at Bb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:43:165)
    at xb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:22:216)
    at yb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:21:1260)
    at Yh.g.Eb (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:206:287)
    at Kh.g.Gd (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:193:283)
    at yh.Gd (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:183:364)
    at qh.xg (/cache/project-www/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:181:281) 
```

**Comment by coffeejunk** [comment]
Same here on wercker ci

```
i  deploying hosting
i  preparing public directory for upload...
FIREBASE WARNING: Exception was thrown by user callback. TypeError: undefined is not a function
    at ProgressBar.terminate (/usr/lib/node_modules/firebase-tools/node_modules/progress/lib/node-progress.js:177:17)
    at ProgressBar.tick (/usr/lib/node_modules/firebase-tools/node_modules/progress/lib/node-progress.js:91:10)
    at /usr/lib/node_modules/firebase-tools/lib/deploy/hosting/deploy.js:49:15
    at /usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:46:642
    at Bb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:43:165)
    at xb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:22:216)
    at yb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:21:1260)
    at Yh.g.Eb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:206:287)
    at Kh.g.Gd (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:193:298)
    at yh.Gd (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:183:364) 

Error: An unexpected error has occurred.
FIREBASE WARNING: Exception was thrown by user callback. TypeError: undefined is not a function
    at ProgressBar.terminate (/usr/lib/node_modules/firebase-tools/node_modules/progress/lib/node-progress.js:177:17)
    at /usr/lib/node_modules/firebase-tools/lib/deploy/hosting/deploy.js:54:15
    at /usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:46:642
    at Bb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:43:165)
    at xb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:22:216)
    at yb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:21:1260)
    at Yh.g.Eb (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:206:287)
    at Kh.g.Gd (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:193:298)
    at yh.Gd (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:183:364)
    at qh.xg (/usr/lib/node_modules/firebase-tools/node_modules/firebase/lib/firebase-node.js:181:281) 

Error: An unexpected error has occurred.
✔  5 files uploaded successfully
```

**Comment by drtriumph** [comment]
Fix incoming on #72 - apologies for the inconvenience

**Comment by mbleigh** [comment]
Closed via #72 and now released as 2.0.1!

**Comment by coffeejunk** [comment]
:sparkles: thanks!
